server/src/main/java/org/apache/iotdb/db/utils/AuthUtils.java

```diff
@@ -188,10 +188,22 @@ public static boolean validatePassword(String originPassword, String encryptPass
    *
    * @param pathA sub-path
    * @param pathB path
-   * @return True if pathA == pathB, or pathA is an extension of pathB, e.g. pathA = "root.a.b.c"
-   *     and pathB = "root.a"
+   * @return True if pathA == pathB, or pathA is an extension of pathB, or pathA is contained in the
+   *     semantics of "*" and "**" in pathB, e.g. pathA = "root.a.b.c" and pathB = "root.a",
+   *     "root.a.b.*" or "root.a.**"
    */
   public static boolean pathBelongsTo(String pathA, String pathB) {
+    if (pathB.endsWith(".*")) {
+      if (pathA.endsWith("`")) {
+        pathA = pathA.substring(0, pathA.lastIndexOf(IoTDBConstant.PATH_SEPARATOR + "`"));
```

Comment:
Why should we process back quote ``? It seems back quote will be removed after sql parse.

Reply:
If the path node contains `.`, the lastIndexOf will get a value we don't want such as root.b.\`c.d\`

Reply:
We use single quote '' or double quote "" to help use * in node name, rather than back quote ``.